portfolio/src/main/webapp/script.js

```diff
@@ -283,9 +295,13 @@ function makeDeleteButton(comment, commentElement) {
 }
 
 /**
+ * When clicked, the edit button needs to modify the comment
+ * element so the client can see that it is currently being
+ * edited, replace the permanent marker with a temporary one,
+ * and get the comment details and put them in the form.
```

Comment:
nit: Another thing you can do with jsdoc is add the types of the parameters that are passed. At Google, we use a tool called JsCompiler to ensure type safety. In the future we plan to migrate to Typescript, which offers type-safe JavaScript transpilation. Example of params:
```
@param {string} author The author's name
@param {string} comment The comment for this edit button
@param {Element} commentElement The DOM element that holds the comment
```

Reply:
Good point, I did it for a few functions but not for all of them so I'll keep this in mind.